packages/lit-dev-content/site/home/5-who-is-using.html

```diff
@@ -151,6 +151,16 @@ <h1>
           <span>Wired Elements</span>
         </div>
       </a>
+
+      <a target="_blank" rel="noopener" href="https://github.com/ithaka/pharos#:~:text=lit-element">
```

Comment:
This is very cool--thanks for the PR! One thing we ask is that the link points to a mention of Lit or LitElement. This could be a mention in the docs, a blog post, or even a pointer to something on GitHub, like this:
```suggestion
      <a target="_blank" rel="noopener" href="https://github.com/ithaka/pharos/blob/main/packages/pharos/CHANGELOG.md#:~:text=Lit">
```
I thought I'd found a section on your docs that mentioned Lit, but I can't find it anymore.

Reply:
Makes sense! We mention in our developer docs [here](https://github.com/ithaka/pharos/blob/70f42ef0776d1b0ff0ea32ae6560deef92757d8d/docs/development/anatomy-of-a-component.md#lit) if that's better.

Reply:
The one currently in the PR points to a `lit-element` tag we have on the repo

Reply:
Perfect! That's what I was looking for.

Reply:
I missed both your second comment and the tag on the repo... But I think the other link is better, anyway. This should be live now.